Source/JavaScriptCore/wasm/WasmFormat.h

```diff
@@ -99,13 +99,22 @@ inline JSString* typeToString(VM& vm, TypeKind type)
     RELEASE_ASSERT_NOT_REACHED();
 }
 
-inline bool isRefType(Type type)
+inline bool isRefType(const Type type)
```

Comment:
Minor: I think the `const` qualifiers are not really necessary here or below since the types are non-reference/pointer.